docs/pages/blog/2020-q3-update.md

```diff
@@ -118,7 +118,7 @@ Here are the most significant improvements since June 2020. This was a dense qua
   <p class="blog-description">Slider powered by styled-components</p>
 
 - ♿︎ We have kept investing in accessibility, we have fixed [13 bugs](https://github.com/mui/material-ui/pulls?q=is%3Apr+label%3Aaccessibility+is%3Aclosed+sort%3Aupdated-desc).
-- 🗓 We have introduced public quarterly roadmaps, both for [MUI Core](https://github.com/orgs/mui/projects/18/views/1) and [MUI X](https://github.com/mui/mui-x/projects/1).
+- 🗓 We have introduced public quarterly roadmaps, both for each [MUI Core](https://github.com/mui/material-ui/projects?query=is%3Aopen) product and [MUI X](https://github.com/mui/mui-x/projects/1).
```

Comment:
Same thing as the first comment — linking to the Projects tab instead of a specific project.